src/nomad_simulations/schema_packages/utils/utils.py

```diff
@@ -168,3 +170,54 @@ def get_composition(children_names: 'list[str]') -> str:
     children_count_tup = np.unique(children_names, return_counts=True)
     formula = ''.join([f'{name}({count})' for name, count in zip(*children_count_tup)])
     return formula if formula else None
+
+
+def check_simulation_cell(cell_1: 'Cell', cell_2: 'Cell') -> bool:
+    """
+    Check if the two `Cell` objects are the same by checking if the defined `positions` are all matching. If
+    the objects are `AtomicCell`, it checks if the `AtomsState[*].chemical_symbol` are the same.
+
+    Args:
+        cell_1 (Cell): The first `Cell` to compare.
+        cell_2 (Cell): The second `Cell` to compare.
+
+    Returns:
+        bool: True if the cells are the same, False otherwise.
+    """
+    # If any of the cells is None or the positions are empty, return False
+    if cell_1 is None or cell_2 is None:
+        return False
+    if cell_1.positions is None or cell_2.positions is None:
+        return False
+
+    # The `positions` should have the same length (same number of positions)
+    if len(cell_1.positions) != len(cell_2.positions):
+        return False
+    n_positions = len(cell_1.positions)
+
+    # Check that all the `positions`` of `cell_1` match with the ones in `cell_2`
+    check_positions = []
+    for i1, pos1 in enumerate(cell_1.positions):
+        for i2, pos2 in enumerate(cell_2.positions):
+            if np.allclose(pos1, pos2):
+                check_positions.append([i1, i2])
+                break
+    if len(check_positions) != n_positions:
```

Comment:
easier to check length so it should go before.

Reply:
Well, this is checking a newly created list. 

What I have is 2 lists of positions for each of the cells -> I store in a list the index at which the first and second positions coincide -> if the new list is of length `n_positions` it means that the both cells contain the same positions -> otherwise, there are some missing positions from one to another cell and I should return False